data/master.yaml

```diff
@@ -0,0 +1,167 @@
+      instruments:
+      - Hilfsmittel sind noch zu definieren.
+      id: 211
+  id: 2
+- title: Tools / Dienste
+  description: Zusammenarbeit und Organisation im Team und alleine.
+  year-calendar-weeks: 
+  compentencies: 
+  id: 4
+- title: UI / Web
+  description: Verstehen und Entwickeln von Webseiten
+  year-calendar-weeks:
+    1.1: 1 32-40
+    1.2: 1 50-10
+    1.3: 3 40-50
+    1.4: 4 40-50
+    2.1: 1 40-50
+    2.2: 2 40-50
+    3.1: 1 50-5
+    3.2: 2 33-39
+  compentencies:
+  - title: User Experience (UX)
+    description: Design, Interaktion und Gebrauchstauglichkeit einer Webseite verbessern.
+    id: 13
+    levels:
+    - goals:
+      - Responsive First Ansatz
+      instruments:
+      - https://getbootstrap.com/
+      id: 132
+    - goals:
+      - Was bedeutet User Experience (UX)?
+      - Design und Layout einer Webseite.
+      - Formulare
+      instruments:
+      - Hilfsmittel sind noch zu definieren.
+      id: 131
+  - title: JavaScript in Webseiten
+    description: Einsatz von JavaScript in einer statischen Webseite.
+    id: 12
+    levels:
+    - goals:
+      - Mit jQuery HTML-Tags in den DOM hinzufügen und bestehende HTML-Tags manipulieren.
+      - Daten mit einem Ajax-Request nachladen.
+      instruments:
+      - https://jquery.com/
+      id: 122
+    - goals:
+      - JavaScript direkt in der HTML-Seite.
+      - JavaScript in eine separate Datei auslagern.
+      - Mit JavaScript einfache click-Handler umsetzen.
+      - Ein Formular validieren und absenden.
+      - 'Entwickler-Tools: Debuggen im Browser.'
+      instruments:
+      - Hilfsmittel sind noch zu definieren.
+      id: 121
+  - title: Statische Webseite
+    description: Aufbau und Gestaltung einer statischen Webseite
+    id: 11
+    levels:
+    - goals:
+      - Responsive Webseite verstehen und umsetzen können.
+      instruments:
+      - https://www.w3schools.com/html/html_responsive.asp
+      id: 112
+    - goals:
+      - CSS-Preprocessor wie Less, Sass, Scss anwenden können.
+      instruments:
+      - http://lesscss.org/
+      - https://sass-lang.com/guide
+      - https://sass-lang.com/documentation/syntax
+      id: 114
+    - goals:
+      - 'HTML: Aufbau einer Webseite verstehen: Tags + Attribute.'
+      - 'HTML: Die wichtigsten HTML-Tags kennen.'
+      - 'CSS: Funktionsweise von Selektoren und Formatierung verstehen.'
+      - 'CSS: Die einfachen Styles anwenden können.'
+      - Trennung zwischen HTML und CSS verstehen.
+      - Einfache Layouts mit HTML + CSS umsetzen können.
+      instruments:
+      - https://www.w3schools.com/html
+      - https://www.w3schools.com/css
+      - https://www.codecademy.com/catalog/language/html-css
+      id: 111
+    - goals:
+      - Verwendung von HTML-Framework wie Bootstrap anwenden können.
+      instruments:
+      - https://getbootstrap.com/
+      id: 113
+  id: 1
+conf:
+- semester:
+  - start: KW33
```

Comment:
start_at_week_nr: 33
finish_at_week_nr: 4

it's pretty common in rails/ruby to use _at suffix for all kind of date, time attrs. In this case it's week nr which suites it for me too.